### DISCUSSION TITLE
Not able to run DAG manually with Airflow version 2.3.3

### DISCUSSION THREAD

**Discussion by uscairflow** [apache/airflow#27604] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### What happened

When manually run a DAG from the dashboard (...8080/home), the DAG is stuck in the queue and never triggered, however, the DAG runs with the scheduled time.  This happens with Airflow version 2.3.3

### What you think should happen instead

We have the exactly the same DAG running Airflow version 2.2.2 and it worked with manual run.

### How to reproduce

Click on the DAG, click on the upper right arrow (Trigger DAG) and the DAG is not running but the status shows its queued.

### Operating System

Rocky Linux

### Versions of Apache Airflow Providers

2.3.3

### Deployment

Other

### Deployment details

Azure Pipeline/Releases

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
It's reproducible. Note that issues are not there to debug problem with your deployent 

You need to look at your deployment and logs and check what happens. Likely you have some resource problem. And this is not a "generic" problem - people were using and triggering DAGs in 2.3.3 for months. Without any specific details that you provide, this is not possible to help  - you need to dig deeper.

You need to look at your logs, talk to the admin of your deployment and try to see why it is happening.

converted into a discussion.

**Reply by uscairflow** [reply]
Is there any parameters in the configuration file that can cause this issue?

**Reply by potiuk** [reply]
Absolutely no idea. What made you think there are? Why do you think there are configuration parameters that can cause the issue?

**Reply by uscairflow** [reply]
Everything is the same between 2 versions of Airflow environments (code, scheduler, deployment method, OS) except the configuration file has different parameters and values since the 2 versions of airflow require different configuration file parameters.  To add to the issue producing steps, the DAG can be run manually only for the 1st time, if you run it right after the 1st one, it is only queued.

**Reply by potiuk** [reply]
Dosn't change a thing. You need to look at logs (incliding logs of your deployment) and see what problems you see. For example it can be your run out of memory to start a new task. But only looking at your deployment logs can tell what's wrong.

It's simply impossible to tell what's wrong with your deployment without you spending time on trying to find out. It's very likely the deployment of yours for previous version of Airflow was enough (in terms of resources) and now it is not. But there is absolutely no data (unless you try to dig-in and provide it) for anyone else by you to check what's wrong.

**Reply by uscairflow** [reply]
When the DAG is queued after manually triggered, there is no log directory created for the DAG (tasks) on the server.  The log entry is empty when you click on the tasks and click the Log menu.

**Reply by potiuk** [reply]
Your deployment logs. not airflow logs. No idea what they are but they should show something. This is a deployment issue most likely not airflow issue.

**Reply by potiuk** [reply]
(this is precisely the sign that there is no airflow logs - this means you should look deeper at your deployment logs and see what problems the deployment has).

**Reply by potiuk** [reply]
For example if you run on K8S and task cannot be started - you will see in kubernetes logs why . But that is something that only you can know about and can act upon. There is absolutely nothing anyone else execptyou having access to your system and knowing how to monitor running apps can do.

**Reply by uscairflow** [reply]
I have compared the files and their sizes between the 2 Airflow servers (different versions) they are exactly the same (DAG files as well as Python files).  What other things can go wrong with the deployment?

**Reply by potiuk** [reply]
Did you check reource usage?

**Reply by uscairflow** [reply]
I ran "top |grep airflow" and both servers show identical results

**Reply by potiuk** [reply]
Look at your deployment logs and check for anomalies.

Lookg @uscairflow . There is no way anyone can "guess" what's wrong in your deployment. If you will not provide any evidences I can only say "NO IDEA".  You need to deal with it and dig deeper and find out what's going wrong.
